x-pack/test/plugin_api_perf/plugins/task_manager_performance/index.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { initRoutes } from './init_routes';
+
+function avg(items) {
```

Comment:
If we ever get to the point of wanting to generate more stats, a former colleague of mine wrote a nice small package with a lot of stat functions, if we don't have anything else handy - https://www.npmjs.com/package/stats-lite

Reply:
ooh, that's nice. *stared*